polygon/sync/service.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/ledgerwatch/erigon-lib/chain"
 	"github.com/ledgerwatch/erigon-lib/common"
 	"github.com/ledgerwatch/erigon-lib/direct"
-	executionclient "github.com/ledgerwatch/erigon/cl/phase1/execution_client"
+	execution_proto "github.com/ledgerwatch/erigon-lib/gointerfaces/execution"
```

Comment:
would suggest to rename to `executionproto` to conform to:
- https://github.com/uber-go/guide/blob/master/style.md#package-names
- https://rakyll.org/style-packages/